### ISSUE TITLE
Detect and advertise wrong depth image resolution

### ISSUE THREAD

**Issue by diegomrt** [RoboticsLabURJC/2019-tfm-omar-garrido#5]
If parameters describing depth image resolution are incorrectly set (by swaping rows and columns, for example, as in the screenshot below), rosify_difodo node just crashes without further information:

![rosify_difodo_fail](https://user-images.githubusercontent.com/35877889/71914748-91328600-317a-11ea-9386-f25662dce4cb.png)

**Comment by omar-ogm** [comment]
Fixed on commit 5351e9050e897a9cc2ab28d509b2c9accb44d928. If this fixed your issue please close it

**Comment by diegomrt** [comment]
Just tested. Now an informative ERROR message shows up. Closing.
```
[ERROR] [1578929583.012653498]: The rows_orig(480) specified is not equal to the image height(720)
```
